src/resources/locales/en.json

```diff
@@ -135,11 +138,11 @@
       },
       "menu": {
         "aboutBook": "About this book",
-        "addLib": "Add this book to the librairy",
+        "addLib": "Import the extract",
         "goBuyBook": "Buy this book",
         "goLoanBook": "Borrow this book",
         "goSubBook": "Subscribe",
-        "addTeaser": "Add extract to the librairy"
+        "addTeaser": "Add extract to the library"
```

Comment:
What's the difference with `"addTeaserToLib": "Add the extract book to library"`?

Reply:
A matter of available space; one is in the "book info" view (thre is space) they other in a more constrained space.

Reply:
okay. thank you.